### DISCUSSION TITLE
How to install pyroxy

### DISCUSSION THREAD

**Discussion by Drasart135** [MHProDev/MHDDoS#593] [Q&A]
How to install pyroxy

**Comment by Gamer-art9044** [comment]
If you are using Termux i recommend to run "pkg install rust" first

**Comment by 20player11** [comment]
If you are running linux like me first make env. by running: python3 -m venv myvenv

("myvenv" is the name you want it to be)

then run: source myvenv/bin/activate

after that: pip  install -r requirements.py

**Comment by darkefx** [comment]
Create a file get_proxies.py in your MHDDoS folder:
->  nano get_proxies.py

paste This Script :

->
----------------------------------------------------------------------------------------------
---
  import requests

proxy_sources = [
    "https://api.proxyscrape.com/v2/?request=getproxies&protocol=http&timeout=3000&country=all",
    "https://api.proxyscrape.com/v2/?request=getproxies&protocol=socks4&timeout=3000&country=all",
    "https://api.proxyscrape.com/v2/?request=getproxies&protocol=socks5&timeout=3000&country=all"
]

all_proxies = []

for url in proxy_sources:
    try:
        print(f"Fetching from {url}")
        res = requests.get(url)
        if res.status_code == 200:
            lines = res.text.strip().split("\n")
            all_proxies.extend(lines)
    except Exception as e:
        print(f"Failed to fetch from {url}: {e}")

(hash) Remove duplicates
unique_proxies = list(set(all_proxies))

with open("proxies.txt", "w") as f:
    f.write("\n".join(unique_proxies))

print(f"Saved {len(unique_proxies)} proxies to proxies.txt")
---
---------------------------------------------------------------------------

After that run :

->   python3 get_proxies.py

You Will Get 1000+ proxies
